source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,284 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+In MongoDB, you can use a text search to retrieve documents that
+contain a term or a phrase in a specified field. A term is a sequence of
+characters that excludes whitespace characters. A phrase is a sequence
+of terms that with any number of whitespace characters.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+In the following sections, you run text searches on the
+``fast_and_furious_movies`` collection. Each section uses ``collection``
```

Comment:
Could you specify "a variable named collection" rather than just monospace? The term "uses" is often indeterminate, so either we should add additional description around it or use a more descriptive word.

Reply:
I think we use monospace for variable names rather than quotes. Can you update that?